data/mods/Magiclysm/monsters/zombified_monsters.json

```
@@ -32,5 +32,36 @@
     "death_function": [ "NORMAL" ],
     "upgrades": { "half_life": 28, "into_group": "GROUP_ZOLF_UPGRADE" },
     "flags": [ "SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "KEENNOSE", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "FILTHY" ]
+  },
+  {
+    "id": "mon_zorc",
+    "type": "MONSTER",
+    "name": { "str": "Zorc" },
+    "description": "This unholy beast looks at you with a piercing stare so strong it might scare your soul away.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "categories": [ "CLASSIC" ],
+    "species": [ "ZOMBIE" ],
+    "volume": "67 L",
+    "weight": "200 kg",
+    "hp": 260,
```

```suggestion
    "hp": 122,
```
This is from comparing feral humans (80 HP) to the maximum HP in the zed-classic.json file (100).